### ISSUE TITLE
Cannot start project because of sbt-js-engine

### ISSUE THREAD

**Issue by michalzubkowicz** [playframework/play-java-seed.g8#108]
When I create new application:
sbt new playframework/play-java-seed.g8

After running sbt run I get:
```
[info] Resolved dependencies
[warn]
[warn] Note: Some unresolved dependencies have extra attributes. Check that these dependencies exist with the requested attributes.
[warn] com.typesafe.sbt:sbt-js-engine:1.2.3 (scalaVersion=2.12, sbtVersion=1.0)
[warn] com.typesafe.sbt:sbt-native-packager:1.5.2 (scalaVersion=2.12, sbtVersion=1.0)
[warn] org.foundweekends.giter8:sbt-giter8-scaffold:0.13.1 (sbtVersion=1.0, scalaVersion=2.12)
[warn] com.lightbend.sbt:sbt-javaagent:0.1.5 (scalaVersion=2.12, sbtVersion=1.0)
[warn] com.typesafe.sbt:sbt-twirl:1.5.1 (scalaVersion=2.12, sbtVersion=1.0)
[warn] com.typesafe.sbt:sbt-web:1.4.4 (scalaVersion=2.12, sbtVersion=1.0)
[error] sbt.librarymanagement.ResolveException: Error downloading com.typesafe.sbt:sbt-js-engine;sbtVersion=1.0;scalaVersion=2.12:1.2.3
[error] Not found
[error] Not found
[error] not found: /home/q/.ivy2/localcom.typesafe.sbt/sbt-js-engine/scala_2.12/sbt_1.0/1.2.3/ivys/ivy.xml
[error] not found: https://repo1.maven.org/maven2/com/typesafe/sbt/sbt-js-engine_2.12_1.0/1.2.3/sbt-js-engine-1.2.3.pom
[error] not found: https://repo.scala-sbt.org/scalasbt/maven-releases/com/typesafe/sbt/sbt-js-engine_2.12_1.0/1.2.3/sbt-js-engine-1.2.3.pom
[error] not found: https://repo.scala-sbt.org/scalasbt/maven-snapshots/com/typesafe/sbt/sbt-js-engine_2.12_1.0/1.2.3/sbt-js-engine-1.2.3.pom
[error] not found: https://repo.typesafe.com/typesafe/ivy-releases/com.typesafe.sbt/sbt-js-engine/1.2.3/ivys/ivy.xml
[error] not found: https://repo.scala-sbt.org/scalasbt/ivy-snapshots/com.typesafe.sbt/sbt-js-engine/1.2.3/ivys/ivy.xml
[error] Error downloading com.typesafe.sbt:sbt-native-packager;sbtVersion=1.0;scalaVersion=2.12:1.5.2
[error] Not found
```

Why this package is required and not published?
If this file is in cache everything is working correctly. 
sbt is installed from sbt website and in recent version

**Comment by mkurz** [comment]
A bit late, but: Actually I think there has to be a problem witih your setup. For me everything is working. Which sbt version are you using / have you installed?

**Comment by michalzubkowicz** [comment]
My sbt is:
sbt version in this project: 1.3.10
sbt script version: 1.7.2

I thought so also that's problem with my configration, but this problem occurred few days ago on my second computer after upgrade. I think it's somehow related to Ubuntu 22.04. I've tried deleting cache directories (.cache, .mvn) without luck. 
I've changed distro from Ubuntu from Arch and on my first computer and everything started to work again so I think it's not my configuration (I've used same home directory on both cases). 

Anyway thanks for the reply ;)
